apps/fyler/src/fyler_server.erl

```diff
@@ -280,6 +279,10 @@ handle_call({cancel_task, TaskId}, _From, State) ->
     end,
   {reply, ok, NewState};
 
+handle_call(current_tasks, _From, #state{managers = Managers} = State) ->
+  Stat = fun(M, _, Stats) -> [{[{M, [fyler_utils:current_task_to_proplist(Task) || Task <- ets:tab2list(M)]}]} | Stats] end,
```

Comment:
Line 283 is too long:   Stat = fun(M, _, Stats) -> [{[{M, [fyler_utils:current_task_to_proplist(Task) || Task <- ets:tab2list(M)]}]} | Stats] end,.